src/main/java/net/coderbot/iris/gui/element/ShaderPackSelectionList.java

```diff
@@ -51,7 +51,7 @@ public void refresh() {
             Iris.logger.error("Error reading files while constructing selection UI", e);
 
             // Not translating this since it's going to be seen very rarely,
-            // We're just trying to get more information on a seemingly untraceable bug:
+            // We're just trying to getTextureUnit more information on a seemingly untraceable bug:
```

Comment:
Uh... find & replace was too aggressive here? :D

Reply:
Ugh.  Didn't turn off the search in text refactor. So annoying.